ports/stm32/boards/stm32f091_af.csv

```diff
@@ -1,90 +1,90 @@
+Port ,Pin ,AF0                                                                                 ,AF1                                                                  ,AF2                                   ,AF3                    ,AF4                           ,AF5                                  ,AF6     ,AF7      ,AF8 ,AF9 ,AF10,AF11,AF12,AF13,AF14,AF15,ADC      
+     ,    ,CAN1/CEC/CRS/EVENTOUT/I2S1/2/IR/RCC/SPI1/2/SYS/TIM1/3/14/15/16/17/TSC/USART1/2/3/4/8,CEC/EVENTOUT/I2C1/2/I2S1/2/IR/SPI1/2/TIM3/15/TSC/USART1/2/3/4/5/6/7/8,EVENTOUT/TIM1/2/16/17/USART5/6/7/8/USB,EVENTOUT/I2C1/TIM15/TSC,CAN1/CRS/I2C1/TIM14/USART3/4/5,I2C2/I2S2/RCC/SPI2/TIM15/16/17/USART6,EVENTOUT,COMP1/2  ,    ,    ,    ,    ,    ,    ,    ,    ,ADC      
+PortA,PA0 ,                                                                                    ,USART2_CTS                                                           ,TIM2_CH1/TIM2_ETR                     ,TSC_G1_IO1             ,USART4_TX                     ,                                     ,        ,COMP1_OUT,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN0 
+PortA,PA1 ,EVENTOUT                                                                            ,USART2_DE/USART2_RTS                                                 ,TIM2_CH2                              ,TSC_G1_IO2             ,USART4_RX                     ,TIM15_CH1N                           ,        ,         ,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN1 
+PortA,PA2 ,TIM15_CH1                                                                           ,USART2_TX                                                            ,TIM2_CH3                              ,TSC_G1_IO3             ,                              ,                                     ,        ,COMP2_OUT,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN2 
+PortA,PA3 ,TIM15_CH2                                                                           ,USART2_RX                                                            ,TIM2_CH4                              ,TSC_G1_IO4             ,                              ,                                     ,        ,         ,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN3 
+PortA,PA4 ,I2S1_WS/SPI1_NSS                                                                    ,USART2_CK                                                            ,                                      ,TSC_G2_IO1             ,TIM14_CH1                     ,USART6_TX                            ,        ,         ,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN4 
+PortA,PA5 ,I2S1_CK/SPI1_SCK                                                                    ,CEC                                                                  ,TIM2_CH1/TIM2_ETR                     ,TSC_G2_IO2             ,                              ,USART6_RX                            ,        ,         ,    ,    ,    ,    ,    ,    ,    ,    ,ADC1_IN5 
```

Comment:
We previously called this `TIMx_CH1_ETR`, but now it's `TIMx_CH1/TIMx_ETR` which will be interpreted differently.

Reply:
https://github.com/micropython/micropython/blob/6117aa686f5faac6692f0def91be8e40a696aca3/ports/stm32/boards/stm32f405_af.csv#L8

Reply:
OK I see... so the F091, specifically for TIM2 has the CH1 and ETR afs on the same pin.

The F091 datasheet (which has a number of stylistic and formatting differences to the others) represents this as `TIM2_CH1_ETR` but everything else (and the xml) represents it as `TIM2_CH1/TIM2_ETR`.

I think the latter is correct... this means we should remove the `CH1_ETR` entry from `SUPPORTED_AFS` in `make-pins.py`, as well as from `pin_defs_stm32.h`.

In a future PR we should consider making this a bit stricter, and each pin should only be allowed to have a single af function per af unit per af index. i.e. you can be SPI1_CLK and I2C2_MISO on AF3, but you can't be both `TIM2_CH1` and `TIM2_ETR`.  As far as I know, this is the only case of that happening, but it would be a good check to add to make-pins.py.